From a15b7c0afef1e7d6a3df7f920ad1db5a87c7a17f Mon Sep 17 00:00:00 2001
From: Gent8 <deboordermounir@gmail.com>
Date: Sun, 19 Oct 2025 15:22:25 +0200
Subject: [PATCH 15/24] Update CHANGELOG.md and refactor theme references to
 replace 'Sunrise' with 'Taison' as the new default theme. Adjust related UI
 components and localization strings accordingly.

---
 CHANGELOG.md                                          |  2 +-
 .../main/java/eu/kanade/domain/ui/UiPreferences.kt    | 11 +----------
 .../main/java/eu/kanade/domain/ui/model/AppTheme.kt   |  2 +-
 .../theme/colorscheme/TachiyomiColorScheme.kt         |  2 +-
 i18n/src/commonMain/moko-resources/am/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/ar/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/as/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/base/strings.xml   |  2 +-
 i18n/src/commonMain/moko-resources/be/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/bg/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/bn/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/ca/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/ceb/strings.xml    |  2 +-
 i18n/src/commonMain/moko-resources/cs/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/cv/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/da/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/de/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/el/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/eo/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/es/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/eu/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/fa/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/fi/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/fil/strings.xml    |  2 +-
 i18n/src/commonMain/moko-resources/fr/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/gl/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/he/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/hi/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/hr/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/hu/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/in/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/it/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/ja/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/jv/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/ka-rGE/strings.xml |  2 +-
 i18n/src/commonMain/moko-resources/kk/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/km/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/kn/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/ko/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/lt/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/lv/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/ml/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/mr/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/ms/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/my/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/nb-rNO/strings.xml |  2 +-
 i18n/src/commonMain/moko-resources/ne/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/nl/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/nn/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/pl/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/pt-rBR/strings.xml |  2 +-
 i18n/src/commonMain/moko-resources/pt/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/ro/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/ru/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/sa/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/sah/strings.xml    |  2 +-
 i18n/src/commonMain/moko-resources/sc/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/sdh/strings.xml    |  2 +-
 i18n/src/commonMain/moko-resources/sk/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/sq/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/sr/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/sv/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/ta/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/te/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/th/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/tr/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/tt/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/uk/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/uz/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/vi/strings.xml     |  2 +-
 i18n/src/commonMain/moko-resources/zh-rCN/strings.xml |  2 +-
 i18n/src/commonMain/moko-resources/zh-rTW/strings.xml |  2 +-
 .../src/main/res/values-night/colors_tachiyomi.xml    |  2 +-
 .../src/main/res/values/colors_tachiyomi.xml          |  2 +-
 74 files changed, 74 insertions(+), 83 deletions(-)

diff --git a/CHANGELOG.md b/CHANGELOG.md
index f741b63b5..defc20119 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -16,7 +16,7 @@ _No unreleased changes._
 ## [v0.19.3] - 2025-10-19
 ### Added
 - Option to scope history entries to the active library category via Settings › Library › Categories.
-- Sunrise theme as the new default look, with the legacy blue palette available as Classic Blue.
+- Taison theme as the new default look, with the legacy blue palette available as Classic Blue.
 
 ### Changed
 - Streamlined the library tab by hiding category tabs by default and adding a toolbar dropdown for quick switching.
diff --git a/app/src/main/java/eu/kanade/domain/ui/UiPreferences.kt b/app/src/main/java/eu/kanade/domain/ui/UiPreferences.kt
index 8d23145e6..7da8e97c1 100644
--- a/app/src/main/java/eu/kanade/domain/ui/UiPreferences.kt
+++ b/app/src/main/java/eu/kanade/domain/ui/UiPreferences.kt
@@ -3,8 +3,6 @@ package eu.kanade.domain.ui
 import eu.kanade.domain.ui.model.AppTheme
 import eu.kanade.domain.ui.model.TabletUiMode
 import eu.kanade.domain.ui.model.ThemeMode
-import eu.kanade.tachiyomi.util.system.DeviceUtil
-import eu.kanade.tachiyomi.util.system.isDynamicColorAvailable
 import tachiyomi.core.common.preference.PreferenceStore
 import tachiyomi.core.common.preference.getEnum
 import java.time.format.DateTimeFormatter
@@ -17,14 +15,7 @@ class UiPreferences(
 
     fun themeMode() = preferenceStore.getEnum("pref_theme_mode_key", ThemeMode.SYSTEM)
 
-    fun appTheme() = preferenceStore.getEnum(
-        "pref_app_theme",
-        if (DeviceUtil.isDynamicColorAvailable) {
-            AppTheme.MONET
-        } else {
-            AppTheme.DEFAULT
-        },
-    )
+    fun appTheme() = preferenceStore.getEnum("pref_app_theme", AppTheme.DEFAULT)
 
     fun themeDarkAmoled() = preferenceStore.getBoolean("pref_theme_dark_amoled_key", false)
 
diff --git a/app/src/main/java/eu/kanade/domain/ui/model/AppTheme.kt b/app/src/main/java/eu/kanade/domain/ui/model/AppTheme.kt
index a31046240..9bcbf771c 100644
--- a/app/src/main/java/eu/kanade/domain/ui/model/AppTheme.kt
+++ b/app/src/main/java/eu/kanade/domain/ui/model/AppTheme.kt
@@ -4,7 +4,7 @@ import dev.icerock.moko.resources.StringResource
 import tachiyomi.i18n.MR
 
 enum class AppTheme(val titleRes: StringResource?) {
-    DEFAULT(MR.strings.theme_sunrise),
+    DEFAULT(MR.strings.theme_taison),
     CLASSIC(MR.strings.theme_classic),
     MONET(MR.strings.theme_monet),
     CATPPUCCIN(MR.strings.theme_catppuccin),
diff --git a/app/src/main/java/eu/kanade/presentation/theme/colorscheme/TachiyomiColorScheme.kt b/app/src/main/java/eu/kanade/presentation/theme/colorscheme/TachiyomiColorScheme.kt
index e9918b091..f14ed2fb9 100644
--- a/app/src/main/java/eu/kanade/presentation/theme/colorscheme/TachiyomiColorScheme.kt
+++ b/app/src/main/java/eu/kanade/presentation/theme/colorscheme/TachiyomiColorScheme.kt
@@ -5,7 +5,7 @@ import androidx.compose.material3.lightColorScheme
 import androidx.compose.ui.graphics.Color
 
 /**
- * Colors for the Sunrise default theme
+ * Colors for the Taison default theme
  * Centered around the accent color #EAB710
  */
 internal object TachiyomiColorScheme : BaseColorScheme() {
diff --git a/i18n/src/commonMain/moko-resources/am/strings.xml b/i18n/src/commonMain/moko-resources/am/strings.xml
index 142ef42f0..8b65ac579 100644
--- a/i18n/src/commonMain/moko-resources/am/strings.xml
+++ b/i18n/src/commonMain/moko-resources/am/strings.xml
@@ -275,6 +275,6 @@
     <string name="backup_restore_missing_trackers">መከታተያዎች አልገቡም:</string>
     <string name="backup_restore_missing_sources">የጠፋ ምንጮች:</string>
     <string name="invalid_backup_file_missing_manga">ምትኬ ማንኛውንም ማንጋ አልያዘም ፡፡</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
 </resources>
diff --git a/i18n/src/commonMain/moko-resources/ar/strings.xml b/i18n/src/commonMain/moko-resources/ar/strings.xml
index 9a47d3951..ebc39e5f3 100644
--- a/i18n/src/commonMain/moko-resources/ar/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ar/strings.xml
@@ -511,7 +511,7 @@
     <string name="theme_yinyang">ين &amp; يانغ</string>
     <string name="theme_tako">تاكو</string>
     <string name="theme_monet">ديناميك</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="about_dont_kill_my_app">تقوم بعض الشركات المصنعة بوضع قيود إضافية على التطبيقات التي قد تقضي على الخدمات التي تعمل في الخلفية. يحتوي هذا الموقع على مزيد من المعلومات حول كيفية إصلاحه.</string>
     <string name="restore_miui_warning">قد لا يعمل النسخ الاحتياطيُّ أو الاستعادة إن عطِّلت أمثَلَة MIUI.</string>
diff --git a/i18n/src/commonMain/moko-resources/as/strings.xml b/i18n/src/commonMain/moko-resources/as/strings.xml
index 32acc87c0..d475860dd 100644
--- a/i18n/src/commonMain/moko-resources/as/strings.xml
+++ b/i18n/src/commonMain/moko-resources/as/strings.xml
@@ -246,7 +246,7 @@
     <string name="theme_light">হালধীয়া</string>
     <string name="theme_dark">গাঢ়</string>
     <string name="theme_monet">ডাইনামিক</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">সেউজ আপেল</string>
     <string name="theme_lavender">লাভেণ্ডাৰ</string>
diff --git a/i18n/src/commonMain/moko-resources/base/strings.xml b/i18n/src/commonMain/moko-resources/base/strings.xml
index b2cdd28b9..dcffa58c8 100644
--- a/i18n/src/commonMain/moko-resources/base/strings.xml
+++ b/i18n/src/commonMain/moko-resources/base/strings.xml
@@ -230,7 +230,7 @@
     <string name="theme_light">Light</string>
     <string name="theme_dark">Dark</string>
     <string name="theme_monet">Dynamic</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_catppuccin">Catppuccin</string>
     <string name="theme_greenapple">Green Apple</string>
diff --git a/i18n/src/commonMain/moko-resources/be/strings.xml b/i18n/src/commonMain/moko-resources/be/strings.xml
index f26efefab..3fc0fd7aa 100644
--- a/i18n/src/commonMain/moko-resources/be/strings.xml
+++ b/i18n/src/commonMain/moko-resources/be/strings.xml
@@ -145,7 +145,7 @@
     <string name="theme_midnightdusk">Паўночнае змярканне</string>
     <string name="theme_greenapple">Зялёны яблык</string>
     <string name="theme_monet">Дынамічная</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Тэма прыкладання</string>
     <string name="theme_dark">Уключаны</string>
diff --git a/i18n/src/commonMain/moko-resources/bg/strings.xml b/i18n/src/commonMain/moko-resources/bg/strings.xml
index 0074717d8..ff0b5c022 100644
--- a/i18n/src/commonMain/moko-resources/bg/strings.xml
+++ b/i18n/src/commonMain/moko-resources/bg/strings.xml
@@ -470,7 +470,7 @@
     <string name="action_show_errors">Вижте пълните подробности за грешката</string>
     <string name="pref_app_theme">Тема</string>
     <string name="theme_monet">Динамично</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">Зелена ябълка</string>
     <string name="theme_midnightdusk">Нощен здрач</string>
diff --git a/i18n/src/commonMain/moko-resources/bn/strings.xml b/i18n/src/commonMain/moko-resources/bn/strings.xml
index 6a2579e34..9b3e88ff3 100644
--- a/i18n/src/commonMain/moko-resources/bn/strings.xml
+++ b/i18n/src/commonMain/moko-resources/bn/strings.xml
@@ -516,7 +516,7 @@
     <string name="theme_midnightdusk">মধ্যরাতের সন্ধ্যা</string>
     <string name="theme_greenapple">সবুজ আপেল</string>
     <string name="theme_monet">প্রগতিশীল</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">অ্যাপ থিম</string>
     <string name="pref_category_appearance">দৃশ্য</string>
diff --git a/i18n/src/commonMain/moko-resources/ca/strings.xml b/i18n/src/commonMain/moko-resources/ca/strings.xml
index 4c87a9d42..7e72a5329 100644
--- a/i18n/src/commonMain/moko-resources/ca/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ca/strings.xml
@@ -509,7 +509,7 @@
     <string name="action_sort_count">Nombre total d’elements</string>
     <string name="pref_app_theme">Tema de l’aplicació</string>
     <string name="theme_monet">Dinàmic</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="action_display_language_badge">Llengua</string>
     <string name="theme_midnightdusk">Capvespre</string>
diff --git a/i18n/src/commonMain/moko-resources/ceb/strings.xml b/i18n/src/commonMain/moko-resources/ceb/strings.xml
index 824250b53..acd2ce8cc 100644
--- a/i18n/src/commonMain/moko-resources/ceb/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ceb/strings.xml
@@ -118,7 +118,7 @@
     <string name="theme_dark">Sa</string>
     <string name="pref_app_theme">Tema sa app</string>
     <string name="theme_monet">Dinamiko</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">Berde nga mansanas</string>
     <string name="theme_midnightdusk">Sa tungang gabii sa kilumkilom</string>
diff --git a/i18n/src/commonMain/moko-resources/cs/strings.xml b/i18n/src/commonMain/moko-resources/cs/strings.xml
index 330354464..7733b68fd 100644
--- a/i18n/src/commonMain/moko-resources/cs/strings.xml
+++ b/i18n/src/commonMain/moko-resources/cs/strings.xml
@@ -508,7 +508,7 @@
     <string name="theme_midnightdusk">Půlnoční Soumrak</string>
     <string name="theme_greenapple">Zelené Jablko</string>
     <string name="theme_monet">Dynamické</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Motiv aplikace</string>
     <string name="label_background_activity">Činnost na pozadí</string>
diff --git a/i18n/src/commonMain/moko-resources/cv/strings.xml b/i18n/src/commonMain/moko-resources/cv/strings.xml
index 1b5ed66b1..71f70a3ce 100644
--- a/i18n/src/commonMain/moko-resources/cv/strings.xml
+++ b/i18n/src/commonMain/moko-resources/cv/strings.xml
@@ -523,7 +523,7 @@
     <string name="update_72hour">Кашни 3 кун</string>
     <string name="theme_tealturquoise">Симӗс кӑвак</string>
     <string name="theme_monet">Куҫӑмлӑ</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="label_stats">Шутлавсем</string>
     <string name="action_copy_to_clipboard">Пайлашу асне ӑт</string>
diff --git a/i18n/src/commonMain/moko-resources/da/strings.xml b/i18n/src/commonMain/moko-resources/da/strings.xml
index fa4b75a3f..b2dc64ecc 100644
--- a/i18n/src/commonMain/moko-resources/da/strings.xml
+++ b/i18n/src/commonMain/moko-resources/da/strings.xml
@@ -130,7 +130,7 @@
     <string name="theme_dark">Til</string>
     <string name="pref_app_theme">App tema</string>
     <string name="theme_monet">Dynamisk</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">Æble grøn</string>
     <string name="theme_midnightdusk">Skumring</string>
diff --git a/i18n/src/commonMain/moko-resources/de/strings.xml b/i18n/src/commonMain/moko-resources/de/strings.xml
index 5888b7173..fe2619d49 100644
--- a/i18n/src/commonMain/moko-resources/de/strings.xml
+++ b/i18n/src/commonMain/moko-resources/de/strings.xml
@@ -510,7 +510,7 @@
     <string name="enhanced_services">Erweiterte Tracker</string>
     <string name="enhanced_tracking_info">Bieten für bestimmte Quellen erweiterte Funktionen an. Einträge werden automatisch getrackt, wenn sie deiner Bibliothek hinzugefügt werden.</string>
     <string name="theme_monet">Dynamisch</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="label_background_activity">Hintergrundaktivität</string>
     <string name="pref_lowest">Niedrigste</string>
diff --git a/i18n/src/commonMain/moko-resources/el/strings.xml b/i18n/src/commonMain/moko-resources/el/strings.xml
index e5ec8093b..82110def1 100644
--- a/i18n/src/commonMain/moko-resources/el/strings.xml
+++ b/i18n/src/commonMain/moko-resources/el/strings.xml
@@ -510,7 +510,7 @@
     <string name="enhanced_tracking_info">Παρέχει βελτιωμένες δυνατότητες για συγκεκριμένες πηγές. Οι καταχωρήσεις παρακολουθούνται αυτόματα όταν προστίθενται στη βιβλιοθήκη σας.</string>
     <string name="enhanced_services">Ενισχυμένοι trackers</string>
     <string name="theme_monet">Δυναμικό</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="label_background_activity">Δραστηριότητα παρασκηνίου</string>
     <string name="pref_lowest">Χαμηλότερη</string>
diff --git a/i18n/src/commonMain/moko-resources/eo/strings.xml b/i18n/src/commonMain/moko-resources/eo/strings.xml
index d44ecb645..e370b0893 100644
--- a/i18n/src/commonMain/moko-resources/eo/strings.xml
+++ b/i18n/src/commonMain/moko-resources/eo/strings.xml
@@ -413,7 +413,7 @@
     <string name="theme_tealturquoise">Bluverda k verdblua</string>
     <string name="theme_strawberrydaiquiri">Fraga dajkirio</string>
     <string name="theme_monet">Dinamika</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_category_tracking">Sekvado</string>
     <string name="action_display_local_badge">Loka fonto</string>
diff --git a/i18n/src/commonMain/moko-resources/es/strings.xml b/i18n/src/commonMain/moko-resources/es/strings.xml
index ba4857c93..3b1873324 100644
--- a/i18n/src/commonMain/moko-resources/es/strings.xml
+++ b/i18n/src/commonMain/moko-resources/es/strings.xml
@@ -483,7 +483,7 @@
     <string name="theme_yinyang">Yin y yang</string>
     <string name="theme_greenapple">Manzana verde</string>
     <string name="theme_monet">Dinámico</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Esquema de colores</string>
     <string name="action_start_downloading_now">Empezar la descarga</string>
diff --git a/i18n/src/commonMain/moko-resources/eu/strings.xml b/i18n/src/commonMain/moko-resources/eu/strings.xml
index 59ecbd9ca..db74ac3f6 100644
--- a/i18n/src/commonMain/moko-resources/eu/strings.xml
+++ b/i18n/src/commonMain/moko-resources/eu/strings.xml
@@ -401,7 +401,7 @@
     <string name="theme_dark">Piztu</string>
     <string name="pref_app_theme">Aplikazioaren gaia</string>
     <string name="theme_monet">Dinamikoa</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">Sagar Berdea</string>
     <string name="pref_date_format">Dataren formatua</string>
diff --git a/i18n/src/commonMain/moko-resources/fa/strings.xml b/i18n/src/commonMain/moko-resources/fa/strings.xml
index bccc5540c..f2ccb7ec0 100644
--- a/i18n/src/commonMain/moko-resources/fa/strings.xml
+++ b/i18n/src/commonMain/moko-resources/fa/strings.xml
@@ -475,7 +475,7 @@
     <string name="action_close">بستن</string>
     <string name="pref_app_theme">تم برنامه</string>
     <string name="theme_monet">پویا</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">سیب سبز</string>
     <string name="theme_midnightdusk">غروب نیمه‌شب</string>
diff --git a/i18n/src/commonMain/moko-resources/fi/strings.xml b/i18n/src/commonMain/moko-resources/fi/strings.xml
index e97e27cf9..2e8e9e25e 100644
--- a/i18n/src/commonMain/moko-resources/fi/strings.xml
+++ b/i18n/src/commonMain/moko-resources/fi/strings.xml
@@ -494,7 +494,7 @@
     <string name="action_show_manga">Näytä manga</string>
     <string name="action_display_cover_only_grid">Kansikuva ruudukko</string>
     <string name="theme_monet">Dynaaminen</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">Vihreä omena</string>
     <string name="label_default">Oletus</string>
diff --git a/i18n/src/commonMain/moko-resources/fil/strings.xml b/i18n/src/commonMain/moko-resources/fil/strings.xml
index 85d191452..5623e766b 100644
--- a/i18n/src/commonMain/moko-resources/fil/strings.xml
+++ b/i18n/src/commonMain/moko-resources/fil/strings.xml
@@ -504,7 +504,7 @@
     <string name="theme_midnightdusk">Hatinggabi</string>
     <string name="theme_greenapple">Berdeng Mansanas</string>
     <string name="theme_monet">Nagbabago</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Tema</string>
     <string name="pref_dark_theme_pure_black">Purong itim</string>
diff --git a/i18n/src/commonMain/moko-resources/fr/strings.xml b/i18n/src/commonMain/moko-resources/fr/strings.xml
index 82738c786..74953d283 100644
--- a/i18n/src/commonMain/moko-resources/fr/strings.xml
+++ b/i18n/src/commonMain/moko-resources/fr/strings.xml
@@ -508,7 +508,7 @@
     <string name="theme_midnightdusk">Crépuscule de minuit</string>
     <string name="theme_greenapple">Pomme verte</string>
     <string name="theme_monet">Dynamique</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Thème de l\'appli</string>
     <string name="about_dont_kill_my_app">Certains fabricants ont mis en place des restrictions supplémentaires sur les applications qui tuent les services d\'arrière-plan. Ce site Web contient plus d\'informations sur la manière de résoudre ce problème.</string>
diff --git a/i18n/src/commonMain/moko-resources/gl/strings.xml b/i18n/src/commonMain/moko-resources/gl/strings.xml
index 77e5db4cb..75f9a3689 100644
--- a/i18n/src/commonMain/moko-resources/gl/strings.xml
+++ b/i18n/src/commonMain/moko-resources/gl/strings.xml
@@ -413,7 +413,7 @@
     <string name="theme_tako">Tako</string>
     <string name="theme_yotsuba">Yotsuba</string>
     <string name="theme_monet">Dinámico</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">Mazá verde</string>
     <string name="theme_midnightdusk">Crepúsculo de medianoite</string>
diff --git a/i18n/src/commonMain/moko-resources/he/strings.xml b/i18n/src/commonMain/moko-resources/he/strings.xml
index 73ea427da..563195f00 100644
--- a/i18n/src/commonMain/moko-resources/he/strings.xml
+++ b/i18n/src/commonMain/moko-resources/he/strings.xml
@@ -361,7 +361,7 @@
     <string name="action_sort_date_added">תאריך הוספה</string>
     <string name="action_sort_chapter_fetch_date">תאריך האחזור של הפרקים</string>
     <string name="theme_monet">דינמי</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_library_update_categories_details">פריט הנמצא בקטגוריית מנועי העדכונים לא יעודכן גם אם הוא נכלל בקטגורייה אחרת שכן מתעדכנת.</string>
     <string name="action_show_manga">הראה פריט</string>
diff --git a/i18n/src/commonMain/moko-resources/hi/strings.xml b/i18n/src/commonMain/moko-resources/hi/strings.xml
index 2405dc397..f1f8c6c5d 100644
--- a/i18n/src/commonMain/moko-resources/hi/strings.xml
+++ b/i18n/src/commonMain/moko-resources/hi/strings.xml
@@ -462,7 +462,7 @@
     <string name="theme_strawberrydaiquiri">स्ट्रॉबेरी</string>
     <string name="theme_greenapple">हरा सेब</string>
     <string name="theme_monet">डायनेमिक</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">एप थीम</string>
     <string name="pref_category_auto_download">ऑटोमैटिक डाउनलोड</string>
diff --git a/i18n/src/commonMain/moko-resources/hr/strings.xml b/i18n/src/commonMain/moko-resources/hr/strings.xml
index 7e38a2850..5484bc32f 100644
--- a/i18n/src/commonMain/moko-resources/hr/strings.xml
+++ b/i18n/src/commonMain/moko-resources/hr/strings.xml
@@ -502,7 +502,7 @@
     <string name="theme_midnightdusk">Ponoćni sumrak</string>
     <string name="theme_greenapple">Zelena jabuka</string>
     <string name="theme_monet">Dinamički</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Tema aplikacije</string>
     <string name="theme_yotsuba">Yotsuba</string>
diff --git a/i18n/src/commonMain/moko-resources/hu/strings.xml b/i18n/src/commonMain/moko-resources/hu/strings.xml
index 4a6be12ed..4c93aecbe 100644
--- a/i18n/src/commonMain/moko-resources/hu/strings.xml
+++ b/i18n/src/commonMain/moko-resources/hu/strings.xml
@@ -317,7 +317,7 @@
     <string name="theme_midnightdusk">Éjfél szürkület</string>
     <string name="theme_greenapple">Zöldalma</string>
     <string name="theme_monet">Dinamikus</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Alkalmazás témája</string>
     <string name="pref_category_appearance">Kinézet</string>
diff --git a/i18n/src/commonMain/moko-resources/in/strings.xml b/i18n/src/commonMain/moko-resources/in/strings.xml
index faa555bf8..3d258bf8e 100644
--- a/i18n/src/commonMain/moko-resources/in/strings.xml
+++ b/i18n/src/commonMain/moko-resources/in/strings.xml
@@ -507,7 +507,7 @@
     <string name="theme_midnightdusk">Senja Tengah Malam</string>
     <string name="theme_greenapple">Hijau Apel</string>
     <string name="theme_monet">Dinamis</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Tema aplikasi</string>
     <string name="label_background_activity">Aktivitas latar belakang</string>
diff --git a/i18n/src/commonMain/moko-resources/it/strings.xml b/i18n/src/commonMain/moko-resources/it/strings.xml
index bc49874df..e415b6c46 100644
--- a/i18n/src/commonMain/moko-resources/it/strings.xml
+++ b/i18n/src/commonMain/moko-resources/it/strings.xml
@@ -514,7 +514,7 @@
     <string name="theme_midnightdusk">Crepuscolo di Mezzanotte</string>
     <string name="theme_greenapple">Mela Verde</string>
     <string name="theme_monet">Dinamico</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Tema applicazione</string>
     <string name="confirm_lock_change">Autenticarsi per confermare le modifiche</string>
diff --git a/i18n/src/commonMain/moko-resources/ja/strings.xml b/i18n/src/commonMain/moko-resources/ja/strings.xml
index 0650bfbe7..b62481e34 100644
--- a/i18n/src/commonMain/moko-resources/ja/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ja/strings.xml
@@ -506,7 +506,7 @@
     <string name="pref_app_theme">アプリテーマ</string>
     <string name="action_start_downloading_now">ダウンロードを開始します</string>
     <string name="theme_monet">ダイナミック</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="about_dont_kill_my_app">一部のスマホメーカーはバックグラウンドサービスを終了する追加のアプリ制限を施しています。修正する方法についてはこのWebサイトをご覧ください。</string>
     <string name="label_background_activity">バックグラウンドアクティビティ</string>
diff --git a/i18n/src/commonMain/moko-resources/jv/strings.xml b/i18n/src/commonMain/moko-resources/jv/strings.xml
index 37b7e72cc..547303894 100644
--- a/i18n/src/commonMain/moko-resources/jv/strings.xml
+++ b/i18n/src/commonMain/moko-resources/jv/strings.xml
@@ -173,7 +173,7 @@
     <string name="theme_system">Tumuti sistem</string>
     <string name="pref_app_theme">Tema aplikasi</string>
     <string name="theme_monet">Dinamis</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">Ijo apel</string>
     <string name="parental_controls_info">Iki ora nyegah ekstensi kang ora resmi utawa salah dilabeli kanggo nampilake konten NSFW (18+) ing aplikasi.</string>
diff --git a/i18n/src/commonMain/moko-resources/ka-rGE/strings.xml b/i18n/src/commonMain/moko-resources/ka-rGE/strings.xml
index 45e0fb013..c6368da42 100644
--- a/i18n/src/commonMain/moko-resources/ka-rGE/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ka-rGE/strings.xml
@@ -414,7 +414,7 @@
     <string name="action_start_downloading_now">დაიწყე ჩამოტვირთვა</string>
     <string name="theme_greenapple">მწვანე ვაშლი</string>
     <string name="theme_monet">დინამიური</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">აპლიკაციის თემა</string>
     <string name="label_warning">გაფრთხილება</string>
diff --git a/i18n/src/commonMain/moko-resources/kk/strings.xml b/i18n/src/commonMain/moko-resources/kk/strings.xml
index 132007740..41b115bf2 100644
--- a/i18n/src/commonMain/moko-resources/kk/strings.xml
+++ b/i18n/src/commonMain/moko-resources/kk/strings.xml
@@ -46,7 +46,7 @@
     <string name="secure_screen">Қолданба экранын қорғау</string>
     <string name="pref_app_theme">Қолданба кейпі</string>
     <string name="theme_monet">Динамикалық</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">Жасыл Алма</string>
     <string name="theme_midnightdusk">Ымырт</string>
diff --git a/i18n/src/commonMain/moko-resources/km/strings.xml b/i18n/src/commonMain/moko-resources/km/strings.xml
index 64b8b4f20..768e3b48d 100644
--- a/i18n/src/commonMain/moko-resources/km/strings.xml
+++ b/i18n/src/commonMain/moko-resources/km/strings.xml
@@ -123,7 +123,7 @@
     <string name="theme_light">បិទ</string>
     <string name="theme_dark">បើក</string>
     <string name="theme_monet">ឌីណាមិច</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">ប៉ោមបៃតង</string>
     <string name="theme_midnightdusk">មីដណាយdusk</string>
diff --git a/i18n/src/commonMain/moko-resources/kn/strings.xml b/i18n/src/commonMain/moko-resources/kn/strings.xml
index 1875f20ab..7c85706c1 100644
--- a/i18n/src/commonMain/moko-resources/kn/strings.xml
+++ b/i18n/src/commonMain/moko-resources/kn/strings.xml
@@ -501,6 +501,6 @@
     <string name="categorized_display_settings">ಪ್ರತಿ ವರ್ಗದ ವಿಂಗಡಣೆ ಮತ್ತು ಪ್ರದರ್ಶನಕ್ಕಾಗಿ ಸೆಟ್ಟಿಂಗ್‌ಗಳು</string>
     <string name="restrictions">ನಿರ್ಬಂಧಗಳು: %s</string>
     <string name="action_start_downloading_now">ಈಗಲೇ ಡೌನ್‌ಲೋಡ್ ಪ್ರಾರಂಭಿಸಿ</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
 </resources>
diff --git a/i18n/src/commonMain/moko-resources/ko/strings.xml b/i18n/src/commonMain/moko-resources/ko/strings.xml
index 2bcf79cd1..cacc30906 100644
--- a/i18n/src/commonMain/moko-resources/ko/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ko/strings.xml
@@ -499,7 +499,7 @@
     <string name="action_show_manga">항목 표시</string>
     <string name="action_desc">내림차순</string>
     <string name="theme_monet">다이내믹</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="action_display_cover_only_grid">표지만 있는 격자</string>
     <string name="action_newest">최신 순</string>
diff --git a/i18n/src/commonMain/moko-resources/lt/strings.xml b/i18n/src/commonMain/moko-resources/lt/strings.xml
index 00ab1c097..9a62d2246 100644
--- a/i18n/src/commonMain/moko-resources/lt/strings.xml
+++ b/i18n/src/commonMain/moko-resources/lt/strings.xml
@@ -69,7 +69,7 @@
     <string name="theme_midnightdusk">Vidurnakčio saulėlydis</string>
     <string name="theme_greenapple">Obuolių žalia</string>
     <string name="theme_monet">Dinamiška</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Programos tema</string>
     <string name="theme_dark">Įjungta</string>
diff --git a/i18n/src/commonMain/moko-resources/lv/strings.xml b/i18n/src/commonMain/moko-resources/lv/strings.xml
index 8f3e48877..031b3c4ac 100644
--- a/i18n/src/commonMain/moko-resources/lv/strings.xml
+++ b/i18n/src/commonMain/moko-resources/lv/strings.xml
@@ -196,7 +196,7 @@
     <string name="ext_installer_pref">Instalētājs</string>
     <string name="ext_installer_shizuku_stopped">Shizuku nedarbojas</string>
     <string name="theme_monet">Dinamisks</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="action_sort_count">Visi ieraksti</string>
     <string name="cancel_all_for_series">Atcelt visu šai sērijai</string>
diff --git a/i18n/src/commonMain/moko-resources/ml/strings.xml b/i18n/src/commonMain/moko-resources/ml/strings.xml
index b9eaabf1c..8f804d80a 100644
--- a/i18n/src/commonMain/moko-resources/ml/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ml/strings.xml
@@ -207,6 +207,6 @@
     <string name="onboarding_permission_notifications_description">ലൈബ്രറി അപ്‌ഡേറ്റുകൾക്കും മറ്റും അറിയിപ്പ് നേടുക.</string>
     <string name="onboarding_permission_ignore_battery_opts">പശ്ചാത്തല ബാറ്ററി ഉപയോഗം</string>
     <string name="onboarding_permission_ignore_battery_opts_description">ദീർഘകാല ലൈബ്രറി അപ്‌ഡേറ്റുകൾ, ഡൗൺലോഡ്, ബാക്കപ്പ് പുനഃസ്ഥാപിക്കൽ എന്നിവയ്ക്കുള്ള തടസ്സങ്ങൾ ഒഴിവാക്കുക.</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
 </resources>
diff --git a/i18n/src/commonMain/moko-resources/mr/strings.xml b/i18n/src/commonMain/moko-resources/mr/strings.xml
index 7b88a6f16..14c523870 100644
--- a/i18n/src/commonMain/moko-resources/mr/strings.xml
+++ b/i18n/src/commonMain/moko-resources/mr/strings.xml
@@ -213,6 +213,6 @@
     <string name="action_set_interval">मध्यांतर</string>
     <string name="action_filter_tracked">ट्रॅक केले</string>
     <string name="action_filter_interval_custom">कस्टमाईज्ड फेच इंटरवल</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
 </resources>
diff --git a/i18n/src/commonMain/moko-resources/ms/strings.xml b/i18n/src/commonMain/moko-resources/ms/strings.xml
index c614d3a66..79735e0ca 100644
--- a/i18n/src/commonMain/moko-resources/ms/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ms/strings.xml
@@ -504,7 +504,7 @@
     <string name="theme_midnightdusk">Midnight Dusk</string>
     <string name="theme_greenapple">Epal Hijau</string>
     <string name="theme_monet">Dinamik</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Tema aplikasi</string>
     <string name="enhanced_tracking_info">Perkidmatan yang menyediakan fungsi yang dipertingkatkan untuk sumber tertentu. Entri akan dijejaki secara automatik apabila ditambahkan ke pustaka.</string>
diff --git a/i18n/src/commonMain/moko-resources/my/strings.xml b/i18n/src/commonMain/moko-resources/my/strings.xml
index bd1172a1b..4b6dd6614 100644
--- a/i18n/src/commonMain/moko-resources/my/strings.xml
+++ b/i18n/src/commonMain/moko-resources/my/strings.xml
@@ -292,6 +292,6 @@
     <string name="network_not_metered">ကန့်သက်မထားသော ကွန်ရက်ပေါ်တွင်သာ</string>
     <string name="ext_uninstall">သင့်ဖုန်းမှ ဖယ်ရှားရန်</string>
     <string name="theme_monet">Dynamic</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
 </resources>
diff --git a/i18n/src/commonMain/moko-resources/nb-rNO/strings.xml b/i18n/src/commonMain/moko-resources/nb-rNO/strings.xml
index 55cd8dd70..5ca5acf4b 100644
--- a/i18n/src/commonMain/moko-resources/nb-rNO/strings.xml
+++ b/i18n/src/commonMain/moko-resources/nb-rNO/strings.xml
@@ -504,7 +504,7 @@
     <string name="restrictions">Begrensninger: %s</string>
     <string name="relative_time_today">I dag</string>
     <string name="theme_monet">Dynamisk</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Programdrakt</string>
     <string name="pref_category_appearance">Utseende</string>
diff --git a/i18n/src/commonMain/moko-resources/ne/strings.xml b/i18n/src/commonMain/moko-resources/ne/strings.xml
index 65b567ee4..e3cb2b619 100644
--- a/i18n/src/commonMain/moko-resources/ne/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ne/strings.xml
@@ -151,7 +151,7 @@
     <string name="theme_midnightdusk">मिडनाइट डस्क</string>
     <string name="theme_greenapple">हरियो स्याउ</string>
     <string name="theme_monet">गतिशील</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">एपको थीम</string>
     <string name="theme_system">सिस्टम पालना गर्नुहोस्</string>
diff --git a/i18n/src/commonMain/moko-resources/nl/strings.xml b/i18n/src/commonMain/moko-resources/nl/strings.xml
index 6b6cfb4f6..da312ba39 100644
--- a/i18n/src/commonMain/moko-resources/nl/strings.xml
+++ b/i18n/src/commonMain/moko-resources/nl/strings.xml
@@ -508,7 +508,7 @@
     <string name="enhanced_tracking_info">Bieden uitgebreidere functionaliteit voor bepaalde bronnen. Het tracken van items gebeurt automatisch wanneer je deze toevoegt aan je bibliotheek.</string>
     <string name="enhanced_services">Verbeterde trackers</string>
     <string name="theme_monet">Dynamisch</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="getting_started_guide">Gids voor beginners</string>
     <string name="help_translate">Help met vertalen</string>
diff --git a/i18n/src/commonMain/moko-resources/nn/strings.xml b/i18n/src/commonMain/moko-resources/nn/strings.xml
index 4aef7a552..5816adbef 100644
--- a/i18n/src/commonMain/moko-resources/nn/strings.xml
+++ b/i18n/src/commonMain/moko-resources/nn/strings.xml
@@ -81,7 +81,7 @@
     <string name="theme_light">Av</string>
     <string name="theme_dark">På</string>
     <string name="theme_monet">Dynamisk</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="information_no_downloads">Ingen nedlastingar</string>
     <string name="information_no_recent">Ingen nylege oppdateringar</string>
diff --git a/i18n/src/commonMain/moko-resources/pl/strings.xml b/i18n/src/commonMain/moko-resources/pl/strings.xml
index e93ee90e6..588385390 100644
--- a/i18n/src/commonMain/moko-resources/pl/strings.xml
+++ b/i18n/src/commonMain/moko-resources/pl/strings.xml
@@ -504,7 +504,7 @@
     <string name="action_start_downloading_now">Zacznij pobierać teraz</string>
     <string name="theme_greenapple">Zielone jabłko</string>
     <string name="theme_monet">Dynamiczny</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Motyw aplikacji</string>
     <string name="restore_miui_warning">Funkcje kopii zapasowej mogą nie działać, jeśli optymalizacja nakładki MIUI jest wyłączona.</string>
diff --git a/i18n/src/commonMain/moko-resources/pt-rBR/strings.xml b/i18n/src/commonMain/moko-resources/pt-rBR/strings.xml
index 4dbf38d6c..1a59bc912 100644
--- a/i18n/src/commonMain/moko-resources/pt-rBR/strings.xml
+++ b/i18n/src/commonMain/moko-resources/pt-rBR/strings.xml
@@ -510,7 +510,7 @@
     <string name="enhanced_tracking_info">Oferecem recursos aprimorados para fontes específicas. Os itens são automaticamente monitorados quando adicionados em sua biblioteca.</string>
     <string name="enhanced_services">Monitoradores aprimorados</string>
     <string name="theme_monet">Dinâmico</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="label_background_activity">Atividade em segundo plano</string>
     <string name="pref_lowest">Mais baixa</string>
diff --git a/i18n/src/commonMain/moko-resources/pt/strings.xml b/i18n/src/commonMain/moko-resources/pt/strings.xml
index d72f68cad..f9ae2d6b3 100644
--- a/i18n/src/commonMain/moko-resources/pt/strings.xml
+++ b/i18n/src/commonMain/moko-resources/pt/strings.xml
@@ -517,7 +517,7 @@
     <string name="theme_tealturquoise">Azul-petróleo e Turquesa</string>
     <string name="relative_time_today">Hoje</string>
     <string name="theme_monet">Dinâmico</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="label_warning">Aviso</string>
     <string name="confirm_lock_change">Autentifique-se para confirmar a mudança</string>
diff --git a/i18n/src/commonMain/moko-resources/ro/strings.xml b/i18n/src/commonMain/moko-resources/ro/strings.xml
index daca426bc..1839e3caf 100644
--- a/i18n/src/commonMain/moko-resources/ro/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ro/strings.xml
@@ -476,7 +476,7 @@
     <string name="action_start_downloading_now">Începe acum descărcarea</string>
     <string name="information_empty_category_dialog">Nu aveți încă nicio categorie.</string>
     <string name="theme_monet">Dinamic</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="label_default">Implicit</string>
     <string name="theme_midnightdusk">Amurg de Miazănoapte</string>
diff --git a/i18n/src/commonMain/moko-resources/ru/strings.xml b/i18n/src/commonMain/moko-resources/ru/strings.xml
index 661e71220..afd39ea3a 100644
--- a/i18n/src/commonMain/moko-resources/ru/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ru/strings.xml
@@ -510,7 +510,7 @@
     <string name="enhanced_tracking_info">Предоставляет расширенные возможности для определённых источников. Серии будут автоматически отслеживаться при добавлении в библиотеку.</string>
     <string name="enhanced_services">Расширенные сервисы отслеживания</string>
     <string name="theme_monet">Динамическая</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="label_background_activity">Фоновая активность</string>
     <string name="pref_lowest">Самая низкая</string>
diff --git a/i18n/src/commonMain/moko-resources/sa/strings.xml b/i18n/src/commonMain/moko-resources/sa/strings.xml
index fe7406388..6cc2da5eb 100644
--- a/i18n/src/commonMain/moko-resources/sa/strings.xml
+++ b/i18n/src/commonMain/moko-resources/sa/strings.xml
@@ -118,7 +118,7 @@
     <string name="theme_system">संविधाम् अनुसरतु</string>
     <string name="theme_light">अशक्तं कुरु</string>
     <string name="theme_monet">गतिशीलम्</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">हरितसेवम्</string>
     <string name="theme_midnightdusk">निशीथसन्ध्या</string>
diff --git a/i18n/src/commonMain/moko-resources/sah/strings.xml b/i18n/src/commonMain/moko-resources/sah/strings.xml
index 8e36d1e73..af9125fa5 100644
--- a/i18n/src/commonMain/moko-resources/sah/strings.xml
+++ b/i18n/src/commonMain/moko-resources/sah/strings.xml
@@ -411,6 +411,6 @@
     <string name="action_display_show_number_of_items">Мал төһө элбэҕин көрдөр</string>
     <string name="action_sort_chapter_fetch_date">Кэлэр күнэ</string>
     <string name="action_filter_tracked">Суолун көрө сылдьабыт</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
 </resources>
diff --git a/i18n/src/commonMain/moko-resources/sc/strings.xml b/i18n/src/commonMain/moko-resources/sc/strings.xml
index 76d0c15ee..dde42e064 100644
--- a/i18n/src/commonMain/moko-resources/sc/strings.xml
+++ b/i18n/src/commonMain/moko-resources/sc/strings.xml
@@ -506,7 +506,7 @@
     <string name="theme_midnightdusk">Intrinada de mesanote</string>
     <string name="theme_greenapple">Mela birde</string>
     <string name="theme_monet">Dinàmicu</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Tema de s\'aplicatzione</string>
     <string name="about_dont_kill_my_app">Unos cantos produtores tenent restritziones de s\'aplicatzione chi istudant sos servìtzios in s\'isfundu. Custu situ web tenet informatziones in prus in subra de comente acontzare su problema.</string>
diff --git a/i18n/src/commonMain/moko-resources/sdh/strings.xml b/i18n/src/commonMain/moko-resources/sdh/strings.xml
index 0f2987cee..686c4129a 100644
--- a/i18n/src/commonMain/moko-resources/sdh/strings.xml
+++ b/i18n/src/commonMain/moko-resources/sdh/strings.xml
@@ -146,7 +146,7 @@
     <string name="theme_dark">داگیرساو</string>
     <string name="pref_app_theme">ڕووکاری پڕۆگرام</string>
     <string name="theme_monet">داینامیک</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">سەوز</string>
     <string name="theme_midnightdusk">مۆر</string>
diff --git a/i18n/src/commonMain/moko-resources/sk/strings.xml b/i18n/src/commonMain/moko-resources/sk/strings.xml
index ed5aabd8a..1871b7e15 100644
--- a/i18n/src/commonMain/moko-resources/sk/strings.xml
+++ b/i18n/src/commonMain/moko-resources/sk/strings.xml
@@ -347,7 +347,7 @@
     <string name="theme_dark">Zap.</string>
     <string name="pref_app_theme">Téma aplikácie</string>
     <string name="theme_monet">Dynamické</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">Zelené jablko</string>
     <string name="theme_yotsuba">Jotsuba</string>
diff --git a/i18n/src/commonMain/moko-resources/sq/strings.xml b/i18n/src/commonMain/moko-resources/sq/strings.xml
index e4156268e..a8c956e40 100644
--- a/i18n/src/commonMain/moko-resources/sq/strings.xml
+++ b/i18n/src/commonMain/moko-resources/sq/strings.xml
@@ -160,7 +160,7 @@
     <string name="theme_strawberrydaiquiri">Daiquiri luleshtrydhe</string>
     <string name="pref_app_theme">Tema e aplikacionit</string>
     <string name="theme_monet">Dinamik</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_tako">Tako</string>
     <string name="pref_date_format">Formati i datës</string>
diff --git a/i18n/src/commonMain/moko-resources/sr/strings.xml b/i18n/src/commonMain/moko-resources/sr/strings.xml
index 997e1c85f..6fce6e132 100644
--- a/i18n/src/commonMain/moko-resources/sr/strings.xml
+++ b/i18n/src/commonMain/moko-resources/sr/strings.xml
@@ -386,7 +386,7 @@
     <string name="action_show_errors">Прикажи детаље</string>
     <string name="pref_category_appearance">Изглед</string>
     <string name="theme_monet">Динамично</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_midnightdusk">Поноћни сумрак</string>
     <string name="theme_strawberrydaiquiri">Даикири јагода</string>
diff --git a/i18n/src/commonMain/moko-resources/sv/strings.xml b/i18n/src/commonMain/moko-resources/sv/strings.xml
index 61ef9184c..ec3b009d0 100644
--- a/i18n/src/commonMain/moko-resources/sv/strings.xml
+++ b/i18n/src/commonMain/moko-resources/sv/strings.xml
@@ -508,7 +508,7 @@
     <string name="theme_midnightdusk">Midnattskymning</string>
     <string name="theme_greenapple">Grönt äpple</string>
     <string name="theme_monet">Dynamisk</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">App-tema</string>
     <string name="about_dont_kill_my_app">Vissa tillverkare har ytterligare appbegränsningar som dödar bakgrundstjänster. Denna webbplats har mer information om hur du åtgärdar det.</string>
diff --git a/i18n/src/commonMain/moko-resources/ta/strings.xml b/i18n/src/commonMain/moko-resources/ta/strings.xml
index de5ef086a..a7087a842 100644
--- a/i18n/src/commonMain/moko-resources/ta/strings.xml
+++ b/i18n/src/commonMain/moko-resources/ta/strings.xml
@@ -251,7 +251,7 @@
     <string name="pref_dark_theme_pure_black">தூய கருப்பு இருண்ட பயன்முறை</string>
     <string name="pref_relative_format">உறவினர் நேர முத்திரைகள்</string>
     <string name="theme_monet">மாறும்</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">பச்சை ஆப்பிள்</string>
     <string name="pref_relative_format_summary">\"%2$s\" க்கு பதிலாக \"%1$s\"</string>
diff --git a/i18n/src/commonMain/moko-resources/te/strings.xml b/i18n/src/commonMain/moko-resources/te/strings.xml
index 76157bea8..76a877576 100644
--- a/i18n/src/commonMain/moko-resources/te/strings.xml
+++ b/i18n/src/commonMain/moko-resources/te/strings.xml
@@ -125,7 +125,7 @@
     <string name="action_install">ప్రతిష్ఠాపించుము</string>
     <string name="action_undo">పూర్వ స్థితికి వెళ్ళుము</string>
     <string name="theme_monet">క్రియాశీలము</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="action_show_errors">తప్పిద వివరములను చూచుటకై నొక్కుము</string>
     <string name="theme_greenapple">ఆకుపచ్చ సేవఫలము</string>
diff --git a/i18n/src/commonMain/moko-resources/th/strings.xml b/i18n/src/commonMain/moko-resources/th/strings.xml
index cb7ab49cb..54e4bb5f4 100644
--- a/i18n/src/commonMain/moko-resources/th/strings.xml
+++ b/i18n/src/commonMain/moko-resources/th/strings.xml
@@ -527,7 +527,7 @@
     <string name="pref_category_appearance">ลักษณะ</string>
     <string name="pref_app_theme">ธีมแอพ</string>
     <string name="theme_monet">ไดนามิก</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="theme_greenapple">กรีนแอปเปิ้ล</string>
     <string name="theme_midnightdusk">มิดไนท์ดัสก์</string>
diff --git a/i18n/src/commonMain/moko-resources/tr/strings.xml b/i18n/src/commonMain/moko-resources/tr/strings.xml
index 5fd8e8e50..c4471eb1b 100644
--- a/i18n/src/commonMain/moko-resources/tr/strings.xml
+++ b/i18n/src/commonMain/moko-resources/tr/strings.xml
@@ -509,7 +509,7 @@
     <string name="theme_midnightdusk">Alacakaranlık</string>
     <string name="theme_greenapple">Yeşil Elma</string>
     <string name="theme_monet">Dinamik</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Uygulama teması</string>
     <string name="label_background_activity">Arka plan etkinliği</string>
diff --git a/i18n/src/commonMain/moko-resources/tt/strings.xml b/i18n/src/commonMain/moko-resources/tt/strings.xml
index fac9e9a0d..a951338af 100644
--- a/i18n/src/commonMain/moko-resources/tt/strings.xml
+++ b/i18n/src/commonMain/moko-resources/tt/strings.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-<string name="theme_sunrise">Sunrise</string>
+<string name="theme_taison">Taison</string>
 <string name="theme_classic">Classic Blue</string>
 </resources>
diff --git a/i18n/src/commonMain/moko-resources/uk/strings.xml b/i18n/src/commonMain/moko-resources/uk/strings.xml
index ed352f0be..61a76ca61 100644
--- a/i18n/src/commonMain/moko-resources/uk/strings.xml
+++ b/i18n/src/commonMain/moko-resources/uk/strings.xml
@@ -514,7 +514,7 @@
     <string name="theme_midnightdusk">Опівнічні Сутінки</string>
     <string name="theme_greenapple">Зелене яблуко</string>
     <string name="theme_monet">Динамічна</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Тема застосунку</string>
     <string name="action_start_downloading_now">Розпочати завантаження зараз</string>
diff --git a/i18n/src/commonMain/moko-resources/uz/strings.xml b/i18n/src/commonMain/moko-resources/uz/strings.xml
index a9bc165bf..9982a0fb0 100644
--- a/i18n/src/commonMain/moko-resources/uz/strings.xml
+++ b/i18n/src/commonMain/moko-resources/uz/strings.xml
@@ -116,7 +116,7 @@
     <string name="action_start_downloading_now">Yuklashni boshlash</string>
     <string name="action_display_comfortable_grid">Qulay ko\'rinish</string>
     <string name="theme_monet">O\'zgaruvchan</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="action_resume">Davom ettirish</string>
     <string name="action_sort_chapter_fetch_date">Oxirgi bob sanasi</string>
diff --git a/i18n/src/commonMain/moko-resources/vi/strings.xml b/i18n/src/commonMain/moko-resources/vi/strings.xml
index 6b3a9661c..2dd5ff13e 100644
--- a/i18n/src/commonMain/moko-resources/vi/strings.xml
+++ b/i18n/src/commonMain/moko-resources/vi/strings.xml
@@ -497,7 +497,7 @@
     <string name="restore_miui_warning">Tính năng tối ưu hóa của MIUI phải được bật lên để việc sao lưu/khôi phục hoạt động tốt.</string>
     <string name="theme_greenapple">Chủ đề: Táo Xanh</string>
     <string name="theme_monet">Chủ đề: Động</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">Chủ đề ứng dụng</string>
     <string name="action_start_downloading_now">Bắt đầu tải xuống ngay</string>
diff --git a/i18n/src/commonMain/moko-resources/zh-rCN/strings.xml b/i18n/src/commonMain/moko-resources/zh-rCN/strings.xml
index 62fdfd806..9d8c0d928 100644
--- a/i18n/src/commonMain/moko-resources/zh-rCN/strings.xml
+++ b/i18n/src/commonMain/moko-resources/zh-rCN/strings.xml
@@ -510,7 +510,7 @@
     <string name="enhanced_tracking_info">为特定图源提供增强功能。作品在添加到书架时会自动同步进度。</string>
     <string name="enhanced_services">增强平台</string>
     <string name="theme_monet">动态</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="label_background_activity">后台活动</string>
     <string name="pref_lowest">最低</string>
diff --git a/i18n/src/commonMain/moko-resources/zh-rTW/strings.xml b/i18n/src/commonMain/moko-resources/zh-rTW/strings.xml
index 5e5a1b191..d6cc7daff 100644
--- a/i18n/src/commonMain/moko-resources/zh-rTW/strings.xml
+++ b/i18n/src/commonMain/moko-resources/zh-rTW/strings.xml
@@ -507,7 +507,7 @@
     <string name="theme_midnightdusk">黃昏</string>
     <string name="theme_greenapple">青蘋果</string>
     <string name="theme_monet">動態</string>
-    <string name="theme_sunrise">Sunrise</string>
+    <string name="theme_taison">Taison</string>
     <string name="theme_classic">Classic Blue</string>
     <string name="pref_app_theme">應用程式主題</string>
     <string name="pref_lowest">最低</string>
diff --git a/presentation-core/src/main/res/values-night/colors_tachiyomi.xml b/presentation-core/src/main/res/values-night/colors_tachiyomi.xml
index 920d79982..270781f44 100644
--- a/presentation-core/src/main/res/values-night/colors_tachiyomi.xml
+++ b/presentation-core/src/main/res/values-night/colors_tachiyomi.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
-  ~ Colors for the Sunrise default theme
+  ~ Colors for the Taison default theme
   ~ Center color: #EAB710
 -->
 <resources>
diff --git a/presentation-core/src/main/res/values/colors_tachiyomi.xml b/presentation-core/src/main/res/values/colors_tachiyomi.xml
index daf7c3d04..22a309b58 100644
--- a/presentation-core/src/main/res/values/colors_tachiyomi.xml
+++ b/presentation-core/src/main/res/values/colors_tachiyomi.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
-  ~ Colors for the Sunrise default theme
+  ~ Colors for the Taison default theme
   ~ Center color: #EAB710
 -->
 <resources>
-- 
2.39.5 (Apple Git-154)

